docs/reference/ml/df-analytics/apis/evaluate-dfanalytics.asciidoc

```diff
@@ -154,16 +154,36 @@ belongs.
   The data type of this field must be categorical.
   
 `predicted_field`::
-  (Required, string) The field in the `index` that contains the predicted value, 
+  (Optional, string) The field in the `index` which contains the predicted value,
   in other words the results of the {classanalysis}.
 
+`results_nested_field`::
+  (Optional, string) The field of the `index` which is a nested array of results.
+
+`predicted_class_name_field`::
+  (Optional, string) The field of the `index` which contains the predicted class name.
+  Must be nested under `results_nested_field`.
+
+`predicted_probability_field`::
+  (Optional, string) The field of the `index` which contains the probability of
+  whether the item belongs to the class in question or not.
+  Must be nested under `results_nested_field`.
+
 `metrics`::
   (Optional, object) Specifies the metrics that are used for the evaluation.
   Available metrics:
 
   `accuracy`:::
     (Optional, object) Accuracy of predictions (per-class and overall).
 
+  `auc_roc`:::
+    (Optional, object) The AUC ROC (area under the curve of the receiver
+    operating characteristic) score and optionally the curve.
+    It is calculated for a particular class X (provided as "class_name")
```

Comment:
```suggestion
    It is calculated for a specific class (provided as "class_name")
```
Small change that might simplify the sentence.

Reply:
Done.